hyperspy/drawing/_markers/point.py

```diff
@@ -70,10 +69,14 @@ class Point(MarkerBase):
 
     def __init__(self, x, y, size=20, **kwargs):
         MarkerBase.__init__(self)
-        lp = {'color': 'black', 'linewidth': None}
+        lp = {'color': 'black'}
         self.marker_properties = lp
         self.set_data(x1=x, y1=y, size=size)
         self.set_marker_properties(**kwargs)
+        # for compatibility with matplotlib 3.1.x - 3.4.x
```

Comment:
is it possible to test for 3.1.x specific functionalities?

Reply:
I'll remove this code. (Default value of zorder=None is already removed.)